### ISSUE TITLE
Ionic storage-angular 3.0.6 cannot load data stored with Ionic storage 2.2.0

### ISSUE THREAD

**Issue by pierresh** [ionic-team/ionic-storage#260]
Hello,

We have an app developed a few years ago which uses Ionic storage 2.2.0. We did a new app version, using Ionic storage-angular 3.0.6. Unfortunately, we found that the data cannot be loaded after we update the app on a device.

However, when we developed in browser, there is no any problem, data are shared between the 2 apps from storage, and we did not do any customization on the storage name/storeName/dbKey.. We do not use SQL lite either.

We did some tests and found that the data of the old app is not deleted and still exists (if we update the new app by the old one just with a higher version number, the data of the old app is loaded properly.)

Do you have any idea where the problem might come from?

Old app
```ts
@NgModule({
  declarations: [
    MyApp,
  ],
  imports: [
    ....
    IonicStorageModule.forRoot(),
    ....
  ],
```

New app
@NgModule({
	declarations: [AppComponent],
	entryComponents: [],
	imports: [
		....
		IonicStorageModule.forRoot(),
		....
  ],
```

**Comment by niekappartmedia** [comment]
@pierresh Did you fix this?

**Comment by pierresh** [comment]
Hello,

Actually no, I think the problem comes from the update of webview as described in that [thread](https://github.com/ionic-team/cordova-plugin-ionic-webview/issues/476).

**Comment by universallistener** [comment]
@pierresh Did you find a fix for this ?

**Comment by pierresh** [comment]
No, we gave up and created a second app, so now we have the old one and the new one separately.
